be/src/storage/rowset/dictcode_column_iterator.cpp

```diff
@@ -3,10 +3,57 @@
 #include "storage/rowset/dictcode_column_iterator.h"
 
 #include "column/column_helper.h"
+#include "gutil/casts.h"
 #include "storage/rowset/scalar_column_iterator.h"
 
 namespace starrocks {
 
+Status GlobalDictCodeColumnIterator::decode_dict_codes(const vectorized::Column& codes, vectorized::Column* words) {
+    const auto& code_column = down_cast<const vectorized::NullableColumn&>(codes).data_column();
+    const auto& code_data = down_cast<vectorized::Int32Column*>(code_column.get())->get_data();
+    const size_t size = code_data.size();
+
+    LowCardDictColumn::Container* container = nullptr;
+    bool output_nullable = words->is_nullable();
+
+    if (output_nullable) {
+        vectorized::ColumnPtr& data_column = down_cast<vectorized::NullableColumn*>(words)->data_column();
```

Comment:
should const null column and const column be take into consideration?

Reply:
In fact, our storage layer does not have to deal with const columns.
But to simplify the logic, I simplify this part of the code with column helper